packages/rpc-graphql/src/__tests__/transaction-tests.ts

```diff
@@ -895,6 +895,168 @@ describe('transaction', () => {
+                                            amount
+                                            decimals
+                                            uiAmount
+                                            uiAmountString
+                                        }
+                                        tokenAmount {
+                                            amount
+                                            decimals
+                                            uiAmount
+                                            uiAmountString
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                `;
+
+                const result = await rpcGraphQL.query(source, { signature });
+                expect(result).toMatchObject({
+                    data: {
+                        transaction: {
+                            message: {
+                                instructions: expect.arrayContaining([
+                                    {
+                                        authority: {
+                                            address: expect.any(String),
+                                        },
+                                        destination: {
+                                            address: expect.any(String),
+                                        },
+                                        feeAmount: {
+                                            amount: expect.any(String),
+                                            decimals: expect.any(Number),
+                                            uiAmount: null, // can't convert decimal to BigInt
+                                            uiAmountString: expect.any(String),
+                                        },
+                                        mint: {
+                                            address: expect.any(String),
+                                        },
+                                        programId: 'TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb',
+                                        source: {
+                                            address: expect.any(String),
+                                        },
+                                        tokenAmount: {
+                                            amount: expect.any(String),
+                                            decimals: expect.any(Number),
+                                            uiAmount: expect.any(BigInt),
+                                            uiAmountString: expect.any(String),
+                                        },
+                                    },
+                                ]),
+                            },
+                        },
+                    },
+                });
+            });
+
+            it('transfer-checked-with-fee-with-multisigAuthority', async () => {
```

Comment:
I think you can probably combine this test and the one above it into the same test. It's exactly the same, just adding one field.

Reply:
Update this test to be combined as I mentioned [here](https://github.com/solana-labs/solana-web3.js/pull/2579#discussion_r1585501070), and then I think this PR is good to go!